src/components/message/Message.jsx

```diff
@@ -13,6 +14,8 @@ const renderIcon = (role, provider) => {
             return (<SimpleIconsOpenai />);
```

Comment:
do we need to check both role and provider? I think we can just directly check `if (provider === "OpenAI") {` and return vs nest them. We can have a `return null;` at the bottom to catch it